src/platform-release.js

```diff
@@ -131,15 +133,15 @@ exports.prepareReleaseBranchCommand = function*() {
     var repos = flagutil.computeReposFromFlag(argv.r);
     var version = flagutil.validateVersionString(argv.version);
     var branchName = getVersionBranchName(version);
-
+    var platform = repos.id;
```

Comment:
this is the wrong way to get platform. Instead get it from the `repo.id` from within the foreach below